src/env0-environments-provider.ts

```diff
@@ -36,6 +36,16 @@ export class Environment extends vscode.TreeItem {
       this.contextValue = "WAITING_FOR_USER";
     }
   }
+
+  setIgnoreLogRestartOnSelect(value: boolean) {
```

Comment:
Since the instance of this class destroyed after every refresh i use static object here